### PR TITLE
Updated Plotting API in Avocado LR

### PR DESCRIPTION
**Pull request by Davidportlouis**
* Updated all plotting API to conform to latest naming convention introduced in this [PR](https://github.com/mlpack/examples/pull/162)
* Updated plotting & data storing directory hierarchy
* Some minor fixes

### CODE REVIEW

**Review by kartikdutt18 — APPROVED** [review]
LGTM!

File: avocado_price_prediction_with_linear_regression/avocado_price_prediction_with_lr_cpp.ipynb [hunk 1]
```diff
@@ -53,7 +53,7 @@
   },
```
Comment (kartikdutt18) [inline avocado_price_prediction_with_linear_regression/avocado_price_prediction_with_lr_cpp.ipynb:53]: <p>I think the metrics should still be printed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlpack/examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (Davidportlouis) [reply avocado_price_prediction_with_linear_regression/avocado_price_prediction_with_lr_cpp.ipynb:53]: I have executed the metrics cell

**Comment by kartikdutt18** [comment]
Thanks for putting this together.

**Pull request merged**